src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -20,7 +20,7 @@ import org.joda.time.DateTime
 object ErrorAggregator {
 
   private val allowedDocTypes = List("main", "crash")
-  private val outputPrefix = "error_aggregates/v1"
+  private val outputPrefix = "error_aggregates/v2"
```

Comment:
Ok let's move this conversation to [bug 1390522](https://bugzilla.mozilla.org/show_bug.cgi?id=1390522).